README.md

```diff
@@ -85,17 +83,24 @@ Usage: putout [options] [path]
 Options
    -h, --help                  display this help and exit
    -v, --version               output version information and exit
-   -f, --format                use a specific output format - default: progress-bar/dump on CI
+   -f, --format                use a specific output format, 
+                               the default is: progress-bar/dump on CI
    -s, --staged                add staged files when in git repository
    --fix                       apply fixes of errors to code
    --fix-count                 count of fixes rounds (defaults to 10)
    --rulesdir                  use additional rules from directory
    --transform                 apply inline transform
    --plugins                   plugins to use splited by comma
-   --enable                    enable rule by name in .putout.json
-   --disable                   disable rule by name in .putout.json
-   --enable-all                enable all rules in .putout.json
-   --disable-all               disable all rules in .putout.json
+   --enable                    edits the project's .putout.json file to
```

Comment:
This is good addition, but this part is a copy of `putout —help` output, and it will be overwritten in next update if `help.json`. Do you suggest to change `putout` output as well? In other we can use table here, similar to `exit codes`

Reply:
Other way to add new part called *Ruler Toggler** and put this information there

Reply:
I amended Help.json (although I kept it all on a single line there). People are probably used to reading help output in this format, but if you'd like I can switch to a table.

Reply:
I like your detailed explanation, but here is how it looks like
![image](https://user-images.githubusercontent.com/1573141/133431309-c1eae99a-1422-47ef-b324-fe419c7d0198.png)

I think the shorter explanation of each command is better because it's simpler to read.

Reply:
Would you prefer it if I were to: a) Shorten it in both places, or b) Shorten it in help.json and convert the README one to a table?

Reply:
I think would be better to shorten in both places, for example:

```diff
-edits the project's .putout.json file to toggle off a specific rule there
+ toggle off rule
```

Or something like this, the thing is `project's .putout.json` I think it's to much information, but we can add some clarifications anyways, for example

```
toggle off rule in "./putout.json"
```

Why don't you like current description:

```
enable rule by name in .putout.json
```

What is wrong with it? Isn't it clear enough? Maybe better would be:

```
enable rule with a name in .putout.json
```

or something like this:

```
  --enable [rule]             enable the rule in ".putout.json" walking up parent directories
```